src/formState.ts

```diff
@@ -567,15 +567,14 @@ function newValueFieldState<T, K extends keyof T>(
     },
 
     blur() {
-      // Now that the user is done editing the field, we sneak in some trim logic
-      if (this._focused) {
-        this.maybeAutoSave();
-      }
+      this.maybeAutoSave();
       this._focused = false;
     },
 
     maybeAutoSave() {
+      // Now that the user is done editing the field, we sneak in some trim logic
```

Comment:
Maybe throw in `is done editing the field (note maybe w/o blurring, i.e. if they hit enter)` b/c I will probably forget this at some point :-D